jvm-runtime/kotlin/scaffolding/bin/.openjdk@17.pkg

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
I'd like a better solution than relying on Hermit for the various language toolchains TBH. I guess at some point FTL should be managing the toolchains itself, but that seems like something for later.